tools/etc/rsyslog.d/45-frr.conf

```diff
@@ -1,6 +1,12 @@
 # The lines below cause all FRR daemons and process to go
 # to /var/log/frr/frr.log, then drops the message so it does
 # not also go to /var/log/syslog, so the messages are not duplicated
+# Set file ownership of the frr.log file so frr can reference it from
+# CLI. Without this, log file is created with root ownership which can
+# cause issues.
+
+$FileOwner frr
```

Comment:
Ok so I just looked at the rsyslogd docs and these are global configuration parameters, which affect all following statements (the doc for `$FileOwner` and `$FileGroup` doesn't say specifically, but `$FileCreateMode` does and I think it's reasonable to assume these 3 work the same way.)

That would mean that the effect of these statements is not limited to `45-frr.conf` (or the `frr.log` file created) but rather extends to **any other logging config loaded after it**, e.g. `50-foobar.conf`.

I'm only 60% confident on this (I use `syslog-ng` personally), but this definitely needs to be confirmed or invalidated.

Reply:
Thanks for the feedback. I'll check into whether this is true or not and find alternatives.